schema/build-config-schema.js

```diff
@@ -498,6 +498,24 @@ for (const rule of rules) {
         }
       };
       break;
+    case "MD054":
+      scheme.properties = {
+        "style": {
+          "description": "Link or image style should be consistent",
```

Comment:
This should say "Link or image style" - I had a few bad examples that included "should be consistent" that I've just now corrected.